pyanaconda/modules/payload/payload.py

```diff
@@ -60,7 +63,9 @@ def process_kickstart(self, data):
         self._payload_handler.process_kickstart(data)
 
     def _create_correct_handler(self, data):
-        if data.liveimg.seen:
+        if conf.system.provides_live_base_image:
+            self._payload_handler = LiveOSHandlerModule()
```

Comment:
This is not necessary, right? Since this handler cannot be requested by kickstart.

Reply:
Yes, this can be removed.

Reply:
Fixed.